src/services/conversionRates/getConversionRatesService.js

```diff
@@ -148,6 +202,18 @@ const getConversionRates = async (app, requestedDate, requestedSymbol = 'ETH') =
 
   const fiat = app.get('fiatWhitelist');
   const stableCoins = app.get('stableCoins') || [];
+  const tokens = app.get('activeTokenWhitelist');
+  let coingeckoId = '';
+
+  const keys = Object.keys(tokens);
```

Comment:
tokens is an array... it's inappropriate? Object.keys or Object.values are used for iterating over objects. Use other standard iterating over array.

Reply:
Not sure what you mean?, will need an example

Reply:
What would be the alternative?